docs/providers/azure/events/http.md

```diff
@@ -0,0 +1,99 @@
+<!--
+title: Serverless Framework - Azure Functions Events - HTTP
+menuText: HTTP
+menuOrder: 1
+description: Setting up HTTP Trigger Events with Azure Functions via the Serverless Framework
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/providers/azure/events/http)
+<!-- DOCS-SITE-LINK:END -->
+
+# HTTP Trigger
+
+Azure Functions has an API endpoint created for each Function App. This service
+allows you to define public HTTP endpoints for your serverless functions.
+
+To create HTTP endpoints as Event sources for your Azure Functions, use the Serverless Framework's easy HTTP Events syntax.
+
+It might be helpful to read the Azure Function [HTTP Trigger docs](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-http-webhook) to learn the full functionality.
+
+## HTTP Trigger Events
+
+### Simple HTTP Endpoint
+
+This setup specifies that the `hello` function should be run when someone accesses the Function App at `api/example/hello` via
+a `GET` request. 
+
+Here's an example:
+
+```yml
+# serverless.yml
+
+functions:
+  example:
+    handler: handler.hello
+    events:
+      - http: true
+        x-azure-settings:
+            name: req #<string>, default - "req", specifies which name it's available on `context.bindings` 
+            methods: #<array> [GET, POST, PUT, DELETE], default - all
+                - GET
+            route: example/hello #<string>, default - <function name>
```

Comment:
It may be important to point out that route parameters can be specified using `{param}`. For example `example/hello/{name}` and is available in `req.params.name`.